src/main/java/com/springboot/star_wars_character_popularity/app/service/VoteService.java

```diff
@@ -1,13 +1,14 @@
 package com.springboot.star_wars_character_popularity.app.service;
 
+import com.springboot.star_wars_character_popularity.app.exception.FileNotSavedException;
 import com.springboot.star_wars_character_popularity.app.model.Vote;
 import org.springframework.web.multipart.MultipartFile;
 
 import java.io.IOException;
 import java.util.List;
 
 public interface VoteService {
-   Vote saveVote(Vote vote, MultipartFile multipartFile) throws IOException;
+   Vote saveVote(Vote vote, MultipartFile multipartFile) throws IOException, FileNotSavedException;
```

Comment:
then you don't need this throws in method declaration. You simply throw Exception